connectors/src/connectors/zendesk/temporal/sync_ticket.ts

```diff
@@ -18,6 +19,39 @@ import type { DataSourceConfig } from "@connectors/types/data_source_config";
 
 const turndownService = new TurndownService();
 
+/**
+ * Deletes a ticket from the db and the data sources.
+ */
+export async function deleteTicket(
```

Comment:
would that file (sync_tickets) belong to `lib` rather than `temporal`?
it seems functions inside aren't temporal-related

Reply:
yes indeed, moving them 👍